### ISSUE TITLE
Sprint Reviews: Ops & R/ZWCO Invitations all fire centres + ceremony format

### ISSUE THREAD

**Issue by tessmgood** [bcgov/wps#1632]
**Describe the task**
Identify and get invitations out to key folks involved in Prep Day in EACH fire centre for our first regular Sprint Review  **January 26**
- need help identifying who all these folks are, as we've been so Kamloops focused (DH / Ops group)
  - R/ZWCO are rostered positions (not a job title)
- some Ops folks have recently retired or moved into TAs, need to refresh the existing list and add to it (DH / Ops group)
- invite them to first regular sprint review with the goal of socializing and workshopping the HFI tool, get their input as we get it over the finish line for March - we need end-users and Ops people (this tool impacts process and procedure - critical time to loop them in)

This work is contingent on NM's Sprint Review format work @nmathava 
- I have a vision from my UX lens - let's collaborate 
  - high level: we focus on one tool at a time, and tell our sprint review invitees which tool and for approx. how long, e.g. we need your input on HFI for the next 4 sprint cycles (tell them in advance) and your attendance is critical to our success 
  - with the right attendees in the room, we go beyond demos to active workshop activities to guide and unblock us, gain buy-in (change management)

In this first regular sprint review back, with new attendees, do we launch the deck BM tasked us with? Separate ticket
- build some energy and awareness around the Agile team, clarify what we do, how we work, and our roadmap

**Acceptance Criteria**
- [ ] first
- [ ] second
- [ ] third

**Additional context**
- Add any other context about the task here.
- Or here
